src/Machine.Specifications.Runner.VisualStudio/Machine.Specifications.Runner.VisualStudio.csproj

```diff
@@ -16,17 +16,17 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
     <DefineConstants>NETSTANDARD</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="[0.10-*, 0.11)" />
-    <PackageReference Include="Machine.Specifications" Version="[0.11.0,2.0.0)" PrivateAssets="All" />
+    <PackageReference Include="Machine.Specifications" Version="1.*" PrivateAssets="All" />
```

Comment:
Revert this please, the range above is what we want.

Reply:
Shouldn't everyone be on v1 of Machine Spec?

Reply:
Not necessarily, we still maintain backwards compatibility.